### DISCUSSION TITLE
[错误]

### DISCUSSION THREAD

**Discussion by yihao1997** [Chia-Network/chia-blockchain#5137] [Support]
debug.log show
2021-05-15T19:42:43.445 full_node chia.full_node.full_node: WARNING  Wasn't able to add transaction with id 749d4502f52f379b51c9d869faaa43fafdc1454400cde4ae3cb64ad40c6d7cd0, status 3 error: Err.DOUBLE_SPEND

**Comment by yihao1997** [comment]
windows 10

**Comment by yihao1997** [comment]
i got 2 PC to plot

**Comment by yihao1997** [comment]
2PC and the same wallet

**Comment by flutech82** [comment]
Same problem, I tryed to send with 0,5 XCH to other wallet but fail because cannot put comma on value, so I put 1 and it seems to go but cannot see any ID and cannot receive any TX id .

**Comment by cccat6** [comment]
No need to worry about DOUBLE SPENDING. It's not related to you and won't affect you.
不要管这个双花的报错，这和你没关系也不会影响你。

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
